src/tcgwars/logic/util/PokemonCardSet.java

```diff
@@ -264,7 +278,7 @@ public void addSpecialCondition(SpecialConditionType type) {
 
   public String getName() {
     try {
-      lastName = getTopPokemonCard().getName();
+      lastName = getTopPokemonCard().getName().replace(" Lv.X", "");
```

Comment:
In-game name for the Pokémon is (specific example) "Empoleon", not "Empoleon Lv.X". Needed for instance when evolving into Empoleon BREAK.

This may need to be rolled back entirely though; the card's name is "Empoleon". Having it be "Empoleon Lv.X" makes it so that cards like Repeat Ball won't be able to pick it up from the deck in all valid cases:

![image](https://user-images.githubusercontent.com/1235066/91665879-1df5cf80-eacf-11ea-8d63-49ad4171c5a9.png)

Reply:
I'd support adding a "Lv.X" inside stuff like the deck builder and the in-client card database though, but that could probably be handled specifically over there.

Reply:
> I'd support adding a "Lv.X" on these cards' name inside stuff like the deck builder and the "Card Database" popup from UI though, but that could probably be handled specifically in those specific cases.

This sounds good. 

Another idea: strip the suffix during the initialization of `Card` instance.